openapi.yaml

```
@@ -303,6 +308,8 @@ paths:
                 phone:
                   type: string
                   format: phone
+                channel:
+                  type: string
```

You can make this an [enum]():

```suggestion
                channel:
                  type: string
                  enum:
                    - sms
                    - whatsapp
```